.storybook/manager.js

```diff
@@ -10,3 +10,7 @@ import theme from './theme';
 addons.setConfig({
   theme,
 });
+
+// Set the Google Analytics tracking ID if the `master` branch is deployed.
+window.STORYBOOK_GA_ID =
```

Comment:
I'm curious, why does the ID setting work here and not in the other file?

Reply:
The addon documentation is lacking in usage guidance and the issue above initially led me to the solution, but as I understand it from https://storybook.js.org/docs/react/configure/overview#configure-story-rendering, `preview.js` is for managing the contents of the `iframe` that renders the stories, while `manager.js` is geared towards handling configuration more specific to Storybook's UI and for setting options, so it made sense to me to place it here.